rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -323,20 +333,50 @@ def _label_id_index_mapping(
     def _invert_mapping(mapping: Dict) -> Dict:
         return {value: key for key, value in mapping.items()}
 
-    def _tag_id_index_mapping(self, training_data: TrainingData) -> Dict[Text, int]:
-        """Create tag_id dictionary"""
+    def _tag_id_index_mappings(
+        self, training_data: TrainingData
+    ) -> Dict[Text, Dict[Text, int]]:
+        """Create tag_id dictionaries for all CRFs."""
 
-        if self.component_config[BILOU_FLAG]:
-            return bilou_utils.build_tag_id_dict(training_data)
+        tag_id_index_mapping_dict = {}
```

Comment:
this names become ridiculous 🙈 is it a mapping to a mapping? Let's come up with a better name